apis/runs_api.py

```diff
@@ -30,17 +32,30 @@ def create_lobby_run(prompt_id, lobby_id):
     )
     return jsonify({"run_id": run_id})
 
+@run_api.post("/quick_runs/<string:prompt_start>/<string:prompt_end>/runs")
+def create_quick_run(prompt_start, prompt_end):
+    run_id = runs.create_quick_run(unquote(prompt_start), unquote(prompt_end), session.get("user_id"))
```

Comment:
what's the need for the unquote thing here? Are there potentially quotes within the title?

Reply:
this is something that I don't know the best way to approach. So essentially I'm passing two strings through the url, and since there are spaces in the articles, i can't put them in directly. So i encoded it when passing in, and the unquote decodes it. its not the greatest, if there is a better way i can fix it